### DISCUSSION TITLE
feat: typeahead/autocomplete/dropdown suggestions for the 'Pull a model from Ollama.com' input box

### DISCUSSION THREAD

**Discussion by dannyl1u** [open-webui/open-webui#7542] [Ideas]
**Is your feature request related to a problem? Please describe.**
This would be a nice-to-have feature for improved UX.  Potentially saves the user time from going to https://ollama.com/models, copy pasting the name of a model and going back to OpenWebUI to pull the model. 

**Describe the solution you'd like**
A typeahead, autocomplete, and/or a suggestion select menu dropdown implemented, which would effectively turn the input box into a search bar, using a hardcoded list of models on the frontend. 

**Describe alternatives you've considered**
Non hard-coded solution, however Ollama.com doesn't have an API that returns a list of the available models to pull, thus we would need to web scrape ollama.com/models to get a list of the models each time

**Additional context**
I have a suggestion select menu dropdown using a hardcoded list of models implemented locally, can make a draft PR but would like to see if anyone has any input or comment on this.

**Comment by ghost** [comment]
It looks like your pull request encompasses "official" models only but it's noteworthy that user submitted models have recently become searchable. If these models are ever supported I'd hope to see a warning of some sort!

**Comment by ghost** [comment]
@tjbck It looks like model pull has been removed, are there plans to restore this functionality?

BTW, the state of the project is fantastic, I'm very grateful. Thank you for everything! 🙏🏻
